srtcore/core.cpp

```diff
@@ -5408,6 +5403,9 @@ bool CUDT::close()
         m_bConnected = false;
     }
 
+    if (m_pCryptoControl)
+        m_pCryptoControl->close();
+
```

Comment:
This is a bugfix for something that has appeared only during testing of the group code. The problem was that sometimes the `close` call wasn't done when required, or sometimes it was attempted to be done twice (after the pointer was cleared or before it was set to a valid object).